### PR TITLE
(FORGE) Specify service restart command for systemd

### PR DESCRIPTION
**Pull request by nkanderson**

### CODE REVIEW

File: manifests/app.pp [hunk 1]
```diff
@@ -103,8 +103,9 @@
       active  => true,
     }
     ~> service { "unicorn_${name}":
-      ensure => running,
-      enable => true,
+      ensure  => running,
+      enable  => true,
+      restart => "/bin/systemctl reload-or-restart unicorn_${name}.service"
```
Comment (yachub) [inline manifests/app.pp:108]: Looks like the new param just needs a trailing comma and it'll be good to go!
```suggestion
      restart => "/bin/systemctl reload-or-restart unicorn_${name}.service",
```

**Review by yachub — CHANGES_REQUESTED** [review]

File: manifests/app.pp (hunk 1 above)
Comment (nkanderson) [reply manifests/app.pp]: Oh gotcha, thanks! I pushed that change, so I'm going to mark this as "ready for review" now.

File: manifests/app.pp [hunk 2]
```diff
@@ -103,8 +103,9 @@
       active  => true,
     }
     ~> service { "unicorn_${name}":
-      ensure => running,
-      enable => true,
+      ensure  => running,
+      enable  => true,
+      restart => "/bin/systemctl reload-or-restart unicorn_${name}.service",
```
Comment (yachub) [inline manifests/app.pp:108]: I think when we migrated to using the modern systemd module, we overlooked that it also handles notifying the service [here](https://github.com/voxpupuli/puppet-systemd/blob/3.0.0/manifests/unit_file.pp#L105), so we should be able to change the [chaining arrow](https://puppet.com/docs/puppet/7/lang_relationships.html#lang_rel_chaining_arrows) from `~>` to `->`, which I believe is causing the double restart.

```puppet
systemd::unit_file { "unicorn_${name}.service":
  content => template('unicorn/systemd/unicorn.service.erb'),
  owner   => 'root',
  group   => '0',
  mode    => '0644',
  enable  => true,
  active  => true,
  restart => "/bin/systemctl reload-or-restart unicorn_${name}.service",
}
-> service { "unicorn_${name}":
  ensure  => running,
  enable  => true,
}
```

**Review by yachub — CHANGES_REQUESTED** [review]

**Comment by nkanderson** [comment]
@yachub nice, looks like this worked! At least as far as the Forge is concerned. Is there anything I need to do to kick off CI?

**Review by yachub — APPROVED** [review]

**Pull request merged**